### ISSUE TITLE
[Discharge Upgrade Wizard] [UX] Determine review screen edit flow for DUW

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-cms#18029]
## Description

Currently on the Discharge Upgrade Wizard 'Review' screen has edit buttons, and if a Veteran edits an answer to a question, it changes the flow and clears all of the answers past that point.

We are wondering if perhaps the Discharge Upgrade Wizard shouldn't have a review screen, or possibly not have edit buttons on the review screen.

## Comments/concerns/information below:

- **Randi May's observation and concern:** Since the questions are each on their own page now, it may not be clear to someone who is editing answers (in DUW) that they've changed the path, and thus are going through more questions after the one they edited 

- **Jordan reviewed and has this comment:** I do think a review screen is helpful, but we may need to rethink the edit buttons. I'm interested in what a11y folks have to say about this.
- Link to [Slack thread](https://dsva.slack.com/archives/C52CL1PKQ/p1714598022469959?thread_ts=1714591633.627719&cid=C52CL1PKQ) with more conversation/information

### User story

**AS A** Veteran who has answered all questions in the Discharge Upgrade Wizard and is on the Review screen
**I WANT** to not be confused by editing an answer to a question, and then having to go back and seemingly need to answer more questions without knowing why
**SO THAT** I am not frustrated by my experience and can complete the wizard and submit with the least amount of confusion.

### Artifacts
- [Design in Figma](https://www.figma.com/file/XHQAQ8ZDKKsDcXfBn5ulhh/Discharge-Upgrade-Wizard?type=design&node-id=0-1&mode=design&t=dByTZbsSvUcxuXe1-0)
- [User flow in Mural ](https://app.mural.co/t/departmentofveteransaffairs9999/m/departmentofveteransaffairs9999/1696352911500/80253b1c7200212cbcbcfb940d2cf4a29fed8417?sender=u9c899abc4fe36c9a698b1647)

## Acceptance criteria

- [x] Update artifacts with new review screen flow
- [x] Updated artifacts are reviewed with product, engineering, and accessibility
- [ ] Reviewed by UX sitewide lead 
- [x] Sent to VA UX lead for sign off 
- [ ] Tag PM to create follow up tickets

**Comment by thejordanwood** [comment]
**End of sprint update:**

This was reviewed in UX sync on 6/27. This is going to roll into next sprint because I still need to implement the feedback I received in that meeting.

- The feedback that was provided suggested moving the alert in my original mockup below the H1 header. It can't be directly below the header because of the way the subtask pattern is coded. I need to come up with a slightly different alert or solution for this. 
- The Figma file needs to be updated with the recent subtask pattern design and design system components.

**Comment by thejordanwood** [comment]
**Update:** 

- I've implemented feedback and updated the Figma file with the latest design system components. 
- I think this is ready for final approvals, if you'd like to take another look @aklausmeier and @davidmpickett

**Comment by aklausmeier** [comment]
I don't need to do a final approval on this one. Let's discuss follow up research in UX sync. 

I would like @davidmpickett to do a copy review on the alerts.

**Comment by davidmpickett** [comment]
I added comments about the alert copy in Figma, but commenting here for PM/DM visibility.

-  I think we could make the conditional alert on forking pages a single line and not need to use the expandable alert. Copy suggestion: "Changing your answer may lead to a new set of questions."
- Added some copy suggestions for the review screen alert. I'm less clear on the logic of when this alert would/wouldn't show, so would be helpful to clarify that before finalizing copy

**Comment by thejordanwood** [comment]
Adding updates and conversations from the Figma file here:

- I've updated the expandable alert headlines to Dave's copy suggestions. I did change the headline to "Changing your answer **will** lead to a new set of questions." since this alert is conditional and will only be shown if changing an answer will definitely lead to more questions. 
- Laura has recommended we keep this as an expandable alert. Regular alerts don't have a tab stop and are less likely to get noticed in a form flow by non-sighted users.
- I believe Chris has clarified the alert logic on the review screen in [this Figma comment](https://www.figma.com/design/XHQAQ8ZDKKsDcXfBn5ulhh?node-id=150-5869#860524282). We can make this alert conditional. 

@davidmpickett If that all sounds good to you, then I believe this ticket it ready to close.
